tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -439,20 +439,16 @@ qos/test_qos_sai.py::TestQosSai::testQosSaiDwrrWeightChange:
       - "asic_type in ['mellanox']"
 
 qos/test_qos_sai.py::TestQosSai::testQosSaiHeadroomPoolSize[None]:
-  xfail:
-    reason: "Image issue on Arista platforms"
-    conditions:
-      - "platform in ['x86_64-arista_7050cx3_32s']"
   skip:
     reason: "Headroom pool size not supported."
     conditions:
       - "hwsku not in ['Arista-7060CX-32S-C32', 'Celestica-DX010-C32', 'Arista-7260CX3-D108C8', 'Force10-S6100', 'Arista-7260CX3-Q64', 'Arista-7050CX3-32S-C32'] and asic_type not in ['mellanox']"
 
 qos/test_qos_sai.py::TestQosSai::testQosSaiHeadroomPoolWatermark[None]:
-  xfail:
```

Comment:
@XuChen-MSFT Should we remove it from tests/common/plugins/conditional_mark/tests_mark_conditions_202012.yaml:

```
qos/test_qos_sai.py::TestQosSai::testQosSaiPgHeadroomWatermark:
  xfail:
    reason: "test issue or image issue, to be RCA'ed"
    conditions:
      - "release in ['202012']"
```

Reply:
Resolved, had removed all qos testcase out from tests_mark_conditions_202012.yaml already.